### PR TITLE
feat: Add manifests for slack-first bot

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/apps/issues/483

### CODE REVIEW

**Review by 4n4nd — COMMENTED** [review]
/lgtm

File: slack-first/base/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      app: slack-first
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: slack-first
+    spec:
+      containers:
+        - name: slack-first
+          image: quay.io/tcoufal/slack-first:latest
```
Comment (anishasthana) [inline slack-first/base/deployment.yaml:20]: Is it too early to move the image over to the operate-first umbrella?

**Review by anishasthana — COMMENTED** [review]
Just had one question. Once that is addressed I'm happy to :ship:

File: slack-first/base/deployment.yaml (hunk 1 above)
Comment (tumido) [reply slack-first/base/deployment.yaml:20]: I think the devel on it is not happening in `operate-first` umbrella - I have my own CI set up on this and I'm releasing to quay from GitHub Actions during A GH release process (see here for example: https://github.com/tumido/slack-first/issues/29). I probably don't want to be releasing to operate-first from a my private (yet public) repo.

**Review by HumairAK — APPROVED** [review]

**Pull request merged**